gravitee-gateway-handlers/gravitee-gateway-handlers-api/src/main/java/io/gravitee/gateway/handlers/api/processor/RequestProcessorChainFactory.java

```diff
@@ -80,10 +87,8 @@ public void afterPropertiesSet() {
             providers.add(new ProcessorSupplier<>(new Supplier<StreamableProcessor<ExecutionContext, Buffer>>() {
                     @Override
                     public StreamableProcessor<ExecutionContext, Buffer> get() {
-                        ApiLoggableRequestProcessor processor = new ApiLoggableRequestProcessor(api.getProxy().getLogging());
-                        // log max size limit is in MB format
-                        // -1 means no limit
-                        processor.setMaxSizeLogMessage((maxSizeLogMessage <= - 1) ? -1 : maxSizeLogMessage * (1024 * 1024));
+                        ApiLoggableRequestProcessor processor = new ApiLoggableRequestProcessor(api.getProxy().getLogging(),
+                                new LogConfiguration(logSSLInfo, logSSLCertificates, (maxSizeLogMessage <= - 1) ? -1 : maxSizeLogMessage * (1024 * 1024)));
```

Comment:
I would prefer to define a `LogConfiguration.Builder` to make it more easier to maintain and manage when adding new configuration properties.

Reply:
fixed